### PR TITLE
HomePage adjustments

### PR DESCRIPTION
**Pull request by curlup**

### CODE REVIEW

File: src/rootStyles.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ export const colors = {
     LIGHT_GREY: "#cfd0d0",
     DARK_BLUE_GREY: "#3b4856",
     logoLightBlue: "#0B9FEC",
-    logoDarkBlue: "#1C65BE"
+    logoDarkBlue: "#1E66BE"
```
Comment (curlup) [inline src/rootStyles.tsx:13]: this one originated as a typo, I believe

Comment (jacoblurye) [reply src/rootStyles.tsx:13]: Ah, nice catch - it actually originated from before I knew there was an SVG version of the logo where we can get the true hexcodes for the logo colors (the above was generated using a program for sampling colors from images). I think `logoLightBlue` might be slightly off as well.

File: src/components/home/HomePage.tsx [hunk 2]
```diff
@@ -158,7 +157,7 @@ const HomePage: React.FunctionComponent<RouteComponentProps> = ({
                             value={
                                 dataOverview
                                     ? filesize(dataOverview.num_bytes, {
-                                          round: 1
+                                          round: 0
```
Comment (jacoblurye) [inline src/components/home/HomePage.tsx:160]: I disagree with this. I don't think terabytes should be rounded to whole numbers, and we'll be in terabyte territory on the portal fairly soon.

Comment (curlup) [reply src/components/home/HomePage.tsx:160]: agreed

File: src/rootStyles.tsx (hunk 1 above)
Comment (curlup) [reply src/rootStyles.tsx:13]: yeah, it's #0C9FEB

File: src/components/home/HomePage.tsx (hunk 2 above)
Comment (curlup) [reply src/components/home/HomePage.tsx:160]: how about always show `"{num_bytes/10^9} GB"` ?

Comment (jacoblurye) [reply src/components/home/HomePage.tsx:160]: What about
```javascript
round: num_bytes < 10e12 ? 0 : 1
```

Comment (curlup) [reply src/components/home/HomePage.tsx:160]: okaya

**Comment by curlup** [comment]
Can't quite figure out why this `aria-labelledby` test fails. I see the "trials" to be rendered there.

**Review by jacoblurye — APPROVED** [review]

**Pull request merged**